Streams 1750 - document new REST api paths (#71)
Axway/streams-open-docs: content/en/docs/subscribers/subscriber-kafka.md

> In order to get an existing subscription, simply do the following GET request:

Reviewer: ```suggestion
To get an existing subscription, use the following GET request:
```

Use the following `GET` request on your topic to get existing subscriptions:

`GET /streams/subscribers/kafka/api/v1/topics/{topicId}/subscriptions`

> See [pagination](/docs/topics-api/#pagination) to get more information about how pagination and sorting work.

Reviewer: ```suggestion
See [pagination](/docs/topics-api/#pagination) for more information on how pagination and sorting work.
```
